libvast/src/system/default_configuration.cpp

```diff
@@ -71,7 +71,7 @@ caf::error default_configuration::merge_root_options(system::application& app) {
   content["vast"].as_dictionary().insert(options.begin(), options.end());
   auto default_fn = caf::defaults::logger::file_name;
   if (caf::get_or(*this, "logger.file-name", default_fn) == default_fn) {
-    path dir = get_or(*this, "vast.dir", defaults::command::directory);
+    path dir = get_or(*this, "vast.directory", defaults::command::directory);
```

Comment:
I'm confused by this change. The command that sets this option uses `dir`: https://github.com/vast-io/vast/blob/master/libvast/src/system/default_application.cpp#L42. I don't see that changed in this commit, so I'd assume the command option no longer gets picked up correctly, no?

Reply:
I remember having discussed this with @tobim at some point. We found that the long option shouldn't be a shortcut and that spelling it out is more intuitive.

Reply:
Oh, I have no objections against the renaming. But I don't see a change to the command option.

Reply:
Ah, agreed, that's still a missing piece.